lib/active_admin/filters/resource_extension.rb

```diff
@@ -174,8 +174,21 @@ def search_status_section
                 else
                   active.filters.each do |filter|
                     li do
-                      span filter.body
-                      b filter.value
+                      if filter.body =~ /ID/i
+                        begin
+                          klass = filter.body.split(' ID').first.gsub(' ','').constantize
```

Comment:
Line is too long. [89/80]<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.